app/models/report.rb

```diff
@@ -50,12 +61,16 @@ def destroy_reported_item
         item.destroy
       end
     end
-    mark_as_reviewed
+    mark_as_reviewed(STATUS_DELETED)
   end
 
-  def mark_as_reviewed
-    Report.where(item_id: item_id, item_type: item_type).update_all(reviewed: true)
+  # rubocop:disable Rails/SkipsModelValidations
+
+  def mark_as_reviewed(with_action=STATUS_NO_ACTION)
+    Report.where(item_id: item_id, item_type: item_type)
+          .update_all(reviewed: true, action: with_action)
```

Comment:
There is no `action` column anymore, did you accidentally drop the migrations?

Reply:
Eagle-eye